CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+### Added
+- added py37 testing. This is currently broken due to `ldna_ssl`; marked this test in `allow_failures`
```

Comment:
Looks like scriptworker's travis attempt is also failing due to PyYaml compiling issues...

Reply:
Added.